test/companies.spec.js

```diff
@@ -37,7 +37,7 @@ describe('Company Integration Tests', () => {
         expect(res).toBeDefined();
         expect(res.contacts.length).toEqual(1);
         expect(res.locations.length).toEqual(1);
-        expect(res.nexus.length).toEqual(3);
+        expect(res.nexus.length).toEqual(2);
```

Comment:
@ted-spence-avalara could you try pushing this change to your 17.12 PR to see if it resolves the build failure? I don't have push access, and the CI authentication is not working with my fork